test/isArray_test.js

```diff
@@ -0,0 +1,54 @@
+goog.require('helper');
```

Comment:
Could you update with goog.module() syntax?

Reply:
Ok I'll do this one and the others, now we require PR #26